src/frontend/src/binder/insert.rs

```diff
@@ -122,7 +122,7 @@ impl Binder {
         let mut target_table_col_idxs: Vec<usize> = vec![];
         'outer: for query_column in &columns {
             let column_name = query_column.real_value();
-            for (col_idx, table_column) in table_source.columns.iter().enumerate() {
+            '_inner: for (col_idx, table_column) in table_source.columns.iter().enumerate() {
```

Comment:
If we don't need, can we remove `'_inner:` ?

Reply:
I will remove it. I added it in the first place, because we also have an `outer` loop.